parser.go

```diff
@@ -7,6 +7,9 @@ import (
 	"strings"
 )
 
+// DefaultValidator is the default validator that is used, if no custom validator is supplied in a Parser.
+var DefaultValidator = NewValidator()
```

Comment:
Perhaps this variable could be unexported or removed? Can I have an example use case for a package write to read or write to this global variable?

Reply:
True, it can be removed.